datapackage.json

```diff
@@ -1,116 +1,80 @@
 {
-  "licenses": [
+  "name": "global-temp",
+  "title": "Global Temperature Time Series",
+  "version": "0.1.0",
+  "license": "ODC-PDDL-1.0",
+  "sources": [
+    {
+      "name": "GISTEMP Global Land-Ocean Temperature Index",
+      "web": "http://data.giss.nasa.gov/gistemp"
+    },
     {
-      "name": "ODC-PDDL-1.0",
-      "path": "http://opendatacommons.org/licenses/pddl/",
-      "title": "Open Data Commons Public Domain Dedication and License v1.0"
+      "name": "Global component of Climate at a Glance (GCAG)",
+      "web": "https://www.ncei.noaa.gov/node/6696"
     }
   ],
-  "name": "global-temp",
   "resources": [
     {
-      "format": "csv",
-      "mediatype": "text/csv",
       "name": "annual",
       "path": "data/annual.csv",
+      "format": "csv",
+      "mediatype": "text/csv",
       "schema": {
         "fields": [
           {
             "name": "Source",
             "type": "string"
           },
           {
-            "description": "YYYY",
             "name": "Year",
-            "type": "year"
+            "type": "date",
```

Comment:
@gradedSystem I don't believe it is actually "date" type as default format would be `YYYY-MM-DD`. You need to either set type as `year` or specify the format.